aten/src/ATen/native/cuda/GroupMMCommon.cuh

```diff
@@ -49,8 +49,8 @@ __global__ void prepare_grouped_gemm_data(
     delta = offs[tid] - start;
     int align = 16 / sizeof(DtypeA);
     CUDA_KERNEL_ASSERT(
-        delta % align == 0 &&
-        "expected dynamic dimension byte size to be multiple of 16 \n");
+        delta >=0 && delta % align == 0 &&
+        "expected dynamic dimension byte size to be non-negative multiple of 16 \n");
```

Comment:
Does this actually print the error message or just `&&` a string with the condition :/?
`CUDA_KERNEL_ASSERT_MSG` seems to be the macro for that?

Reply:
CUDA_KERNEL_ASSERT_MSG is a sugar over this